docker-images/kafka/scripts/kafka_config_generator.sh

```diff
@@ -19,10 +19,23 @@ LISTENER_SECURITY_PROTOCOL_MAP="REPLICATION:SSL"
 SASL_ENABLED_MECHANISMS=""
 
 if [ "$KAFKA_CLIENT_ENABLED" = "TRUE" ]; then
-  LISTENERS="${LISTENERS},CLIENT://0.0.0.0:9092"
```

Comment:
I think these two lines should stay where they were.
* The listener should be always called CLIENT regardless of the authentication type
* It needs to be set for every listeners the same.